src/Halogen/HTML/Properties.purs

```diff
@@ -148,10 +148,10 @@ charset :: forall r i. String -> IProp (charset :: String | r) i
 charset = prop (PropName "charset")
 
 class_ :: forall r i. ClassName -> IProp (class :: String | r) i
-class_ = prop (PropName "className") <<< unwrap
```

Comment:
We can use pattern-matching here to avoid the call to `unwrap`, rather than `unsafeCoerce`